Tasks/Xcode/xcodebuild2.js

```diff
@@ -83,6 +87,10 @@ function processInputs() {
 	xcb.arg('-configuration');
 	xcb.arg(tl.getInput('configuration', true));
 	
+	//Test Results publish inputs
+	publishResults = (tl.getInput('publishJUnitResults', true) == "true");
+    testResultsFiles = tl.getInput('xctoolReporter', false).split(":")[1].trim();
```

Comment:
You should probably guard the [1] as well.  That will null ref on the trim() call
```
'foo'.split(':')[1].trim()
```